packages/office-ui-fabric-react/src/components/Nav/examples/Nav.FabricDemoApp.Example.tsx

```diff
@@ -13,17 +13,20 @@ export const NavFabricDemoAppExample: React.StatelessComponent = () => {
             {
               key: 'ActivityItem',
               name: 'ActivityItem',
-              url: '#/examples/activityitem'
+              url: '#/examples/activityitem',
```

Comment:
These ones shouldn't open in a new tab, since they're within the app.

Also, would you please add this line to `fabric-website/src/SiteDefinition/SiteDefinition.tsx` in the `redirects` array, so that the examples work on the new website?
```
{ from: '#/examples', to: '#/controls/web' }
```